### PR TITLE
Fix splice batch truncation.

### PR DESCRIPTION
**Pull request by rmmh**
Also add a script to bump versions, because it's a tedious and error-prone manual process.

### CODE REVIEW

**Comment by k8s-oncall** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1245)

File: prow/bump.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+if [ "$#" -ne 1 ]; then
+  echo "usage: $0 [program]"
+  exit 1
+fi
+
+cd $(dirname $0)
+
+makefile_version_re="^\(${1^^}_VERSION.*=\s*\)"
+version=$(sed -n "s/$makefile_version_re//p" Makefile)
```
Comment (fejta) [inline prow/bump.sh]: BOO. Just freaking edit the file. The complexity of this script is not work the microscopic time savings.

File: prow/cmd/splice/main.go [hunk 2]
```diff
@@ -41,6 +41,7 @@ var (
 	repoName       = flag.String("repo", "kubernetes", "Repo name")
 	logJson        = flag.Bool("logjson", false, "output log in JSON format")
 	jobConfigs     = flag.String("job-config", "/etc/jobs/jobs", "Where the job-config configmap is mounted.")
+	maxBatchSize   = flag.Int("batch-size", 5, "Maximum batch size")
```
Comment (fejta) [inline prow/cmd/splice/main.go]: Why do we need this as a flag?

File: prow/bump.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (spxtr) [inline prow/bump.sh]: boilerplate

File: prow/bump.sh (hunk 1 above)
Comment (fejta) [inline prow/bump.sh]: https://github.com/kubernetes/test-infra/blob/master/jenkins/e2e-image/Makefile#L16

**Pull request merged**